core/src/main/java/org/ironjacamar/core/connectionmanager/pool/AbstractPool.java

```diff
@@ -397,4 +422,49 @@ else if (txCM.getTransactionIntegration().isConnectableResource(mc))
     * @return The instance
     */
    protected abstract ManagedConnectionPool createManagedConnectionPool(Credential credential);
+
+   /**
+    * Prefill the connection pool
+    *
+    * @param credential the pool credential
+    */
+   @Override
+   public void prefill()
+   {
+
+      if (poolConfiguration.isPrefill())
+      {
+         ManagedConnectionPool mcp = pools.get(getPrefillCredential());
+
+         if (mcp == null)
+         {
+            getManagedConnectionPool(getPrefillCredential());
```

Comment:
it's just to trigger mcp creation. I'm not assigning it because I don't need to do anything else on just created mcp